Cargo.toml

```diff
@@ -40,6 +40,7 @@ derive-new = "0.5.8"
 ctrlc = "3.1"
 lazy_static = "1.4.0"
 thiserror = "1.0"
+metrics = { version = "0.12.1", features = ["std"] }
```

Comment:
[`std`](https://github.com/metrics-rs/metrics/blob/main/metrics/Cargo.toml#L45) is a default feature, you don't need to specify it

Reply:
I don't know why, but it doesn't find `set_boxed_recorder` method without explicit `std` feature. We have to use boxed recorder because we use tcp connection to send metrics, so we build our recorder in runtime (it's possible to use static recorder for simple LogRecorder for example, that's not our case). In metrics lib exporters examples tcp exporter also implements boxed recorder instead of static one.

Reply:
Ok, you are right